### PR TITLE
create privacy content page

### PR DESCRIPTION
**Pull request by shanu798**

### CODE REVIEW

File: src/routes/RootNavigator.tsx [hunk 1]
```diff
@@ -11,30 +9,31 @@ import { VerificationPage } from '../features/verification/VerificationPage';
...
 
 const styles = {
   container: 'evir__root-navigator__container',
 };
 
-export const RootNavigator = () => {
-  const location = useLocation();
-  const navigator = useNavigate();
+export const RootNavigator = () =>
+// const location = useLocation();
```
Comment (vaibhavg15) [inline src/routes/RootNavigator.tsx:21]: Why have you commented all this out?

File: src/features/privacy-content/PrivacyContentPage.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    container: 'evir__privacy-content-page__header-container',
+    title: 'evir__privacy-content-page__header-title',
+    description: 'evir__privacy-content-page__header-description',
+  },
+};
+
+const PrivacyContentPage = () => {
+  const getBackButton = () => (
+    <BackButton
+      className={styles.backButton}
```
Comment (vaibhavg15) [inline src/features/privacy-content/PrivacyContentPage.tsx:27]: OnClick missing.

**Review by vaibhavg15 — CHANGES_REQUESTED** [review]

File: src/routes/RootNavigator.tsx [hunk 3]
```diff
@@ -11,30 +9,31 @@ import { VerificationPage } from '../features/verification/VerificationPage';
...
+//   () => {
+//     if (!(location.pathname === ROUTES.LANDING.path || location.pathname === '/')) {
+//       navigator(
+//         ROUTES.LANDING.path, {
+//           replace: true,
+//         },
+//       );
+//     }
+//   }, [], // do not pass any dependency. This should run only on first render and not on every render
+// );
```
Comment (rohan-1898) [inline src/routes/RootNavigator.tsx:34]: No need to push these changes

File: src/routes/Routes.ts [hunk 4]
```diff
@@ -53,4 +53,10 @@ export const ROUTES = {
     path: '/error',
     name: 'Error',
   },
+  PRIVACY: {
+    id: 'privacy',
+    route: 'privacy',
+    path: '/privacy',
+    name: 'Privacy',
+  },
```
Comment (rohan-1898) [inline src/routes/Routes.ts:61]: we can change these names to customer policy

**Review by rohan-1898 — CHANGES_REQUESTED** [review]

**Pull request merged**